backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Norm.java

```diff
@@ -124,6 +131,18 @@ public Optional<Proprietary> getProprietary() {
     return NodeParser.getNodeFromExpression("//meta/proprietary", document).map(Proprietary::new);
   }
 
+  /**
+   * Returns a {@link Meta} instance from a {@link Document} in a {@link Norm}.
+   *
+   * @return the meta node as {@link Meta}
+   */
+  public Meta getMeta() throws MandatoryNodeNotFound {
+    final String xpath = "//act/meta";
+    return NodeParser.getNodeFromExpression(xpath, document)
+        .map(node -> new Meta(node, getEli()))
+        .orElseThrow(() -> new MandatoryNodeNotFound(xpath));
```

Comment:
I like it. I will do it. We then need to pass the xpath, the document or node and for sure the eli so that we can't then throw the custom exception in the NodeParser, you agree? If so I would implement it

Reply:
sounds good :)

Reply:
Done